quantcast/quantcast-htb-system-tests.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+function getPartnerId() {
+    return 'QuantcastHtb';
+}
+
+function getStatsId() {
+    return 'QUA';
+}
+
+function getBidRequestRegex() {
+    return {
+        method: 'POST',
+        urlRegex: /qcx\.quantserve\.com:8...\/qchb/
+    };
+}
+
+function getCallbackType() {
+    return 'NONE';
+}
+
+function getArchitecture() {
+    return 'MRA';
+}
```

Comment:
Can't we use Partner.CallbackTypes.NONE and Partner.Architectures.MRA here?

Reply:
No. We can't do that. `Partner.CallbackTypes.NONE` and `Partner.Architectures.MRA` are numbers whereas these two functions must return string.